### PR TITLE
medium: filter exist args

### PR DESCRIPTION
**Pull request by liangxin1300**
Apply this commit to avoid this completion situation:

1 primitive vip ocf:Heartbeat:IPaddr2 params ip=10.10.10.123 nic=eth1 nic=xxxx ip=xxxx
2 primitive vip ocf:Heartbeat:IPaddr2 meta is-managed=xxx is-managed=xxx

### CODE REVIEW

File: crmsh/utils.py [hunk 1]
```diff
@@ -42,6 +42,14 @@ def to_ascii(s):
...
 
 
+def completed(item, args, use='='):
+    """Check whether item has been completed in args"""
+    for arg in args:
+        if arg.startswith(item+use):
+            return True
+    return False   
+
+
```
Comment (krig) [inline crmsh/utils.py]: how about a more general function `any_startswith`:

```
def any_startswith(iterable, prefix):
    """Return first element in iterable which startswith prefix, or None."""
    for element in iterable:
        if element.startswith(prefix):
            return element
    return None
```

and then use like this:

```
return [s+'=' for s in constants.rsc_meta_attributes if utils.any_startswith(args, s + '=') is None]
```

**Comment by liangxin1300** [comment]
Thank you, it looks better:)

**Comment by liangxin1300** [comment]
Hi krig, I've changed the code in local days ago, but I forgot to push it:)

**Comment by krig** [comment]
Looks good, thanks :)

**Pull request merged**